Source/FolioReaderPage.swift

```diff
@@ -425,10 +425,6 @@ public class FolioReaderPage: UICollectionViewCell, UIWebViewDelegate, UIGesture
     // MARK: UIMenu visibility
     
     override public func canPerformAction(action: Selector, withSender sender: AnyObject?) -> Bool {
-		guard readerConfig.useReaderMenuController else {
```

Comment:
This can I think you should not remove the guard, because it will call the create menu, the idea is just to call super instead inside the guard.

For customization you can override this without problems.